user-management/src/app/manage/create-user/create-user.component.html

```diff
@@ -0,0 +1,109 @@
+<form [formGroup]="userForm" (ngSubmit)="onSubmit()">
+  <div class="form-group">
+    <label for="firstname">First Name</label>
+    <input
+      type="text"
+      id="firstname"
+      class="form-control"
+      formControlName="firstName"
+    />
+    <span
+      *ngIf="
+        userForm?.get('firstName')?.errors?.required &&
+        userForm?.get('firstName').touched
+      "
+      class="text-danger"
+    >
+      enter first name
+    </span>
+  </div>
+  <div class="form-group">
+    <label for="lastname">Last Name</label>
+    <input
+      type="text"
+      id="lastname"
+      class="form-control"
+      formControlName="lastName"
+    />
+    <span
+    *ngIf="
+      userForm?.get('lastName')?.errors?.required &&
+      userForm?.get('lastName').touched
+    "
+    class="text-danger"
+  >
+    enter last name
+  </span>
+  </div>
+  <div class="form-group">
+    <label for="login">Login</label>
+    <input
+      type="text"
+      id="login"
+      class="form-control"
+      formControlName="login"
+    />
+    <span
+    *ngIf="
+      userForm?.get('login')?.errors?.required &&
+      userForm?.get('login').touched
+    "
+    class="text-danger"
+  >
+    enter login ID
+  </span>
+  </div>
+  <div class="form-group">
+    <label for="password">Password</label>
+    <input
+      type="password"
+      id="password"
+      class="form-control"
+      formControlName="password"
+    />
+    <span
+      *ngIf="
+        userForm?.get('password')?.errors?.required &&
+        userForm?.get('password').touched
+      "
+      class="text-danger"
+    >
+      enter password
```

Comment:
don't hardcore any values or plain text in html pages
maintain a constant file and fetch them from their